Read through

Nikoleta-v3 opened the pull request:
I have now read over the manuscript @drvinceknight.

You can just push on this `PR` your suggestions and then we merge this.

Nikoleta-v3 committed:
small tweaks

Nikoleta-v3 committed:
last rewrites by n.g.

Nikoleta-v3 commented:
commit 9630e09 corrects some minor typos and issues on the manuscript.

commit 95a0884 adds to the abstract, add a sentence about adaptability in the limitations of extortionate behaviour, re writes the conclusion and fixes the appendix on noise

drvinceknight commented on main.tex:
```diff
@@ -59,14 +59,19 @@
     memory in multi agent interactions and can be out performed by
     optimised strategies with longer memory.
+    We have developed a theory that has allowed to explore the the entire
+    space of memory-one strategies. The frame work presented in suitable to
```
```suggestion
    space of memory-one strategies. The framework presented is suitable to
```

drvinceknight commented on main.tex:
```diff
@@ -59,14 +59,19 @@
     optimised strategies with longer memory.
+    We have developed a theory that has allowed to explore the the entire
+    space of memory-one strategies. The frame work presented in suitable to
+    study memory-one strategies in the inifite Prisoner's Dilemma, but to also
+    study evolutionary processes such as Moran process, the Prisoner's Dilemma
+    with noise and the stability of defection in populations of memory-one strategies.
```
```suggestion
    study memory-one strategies in the Prisoner's Dilemma, but also
    in evolutionary processes such as the Moran process,
    Furthermore, results on the stability of defection in populations of 
    memory-one strategies are also obtained.
```

drvinceknight commented on main.tex:
```diff
@@ -415,9 +420,10 @@ \subsection{Limitations of extortionate behaviour} %TODO check that this is a go
 response is not uniformly a ZDs. A positive measure of skewness and kurtosis,
 and a mean of 0.34 indicate a heavy tail to the right. Therefore, in several
-cases the strategy is not trying to extort its opponents.
+cases the strategy is not trying to extort its opponents. This exhibit that
+the best response strategy's behaviour is more adaptable than that of a ZDs.
```
```suggestion
cases the strategy is not trying to extort its opponents. 
```

drvinceknight commented on main.tex:
```diff
@@ -631,34 +635,33 @@ \section{Discussion}
-Note that Theorem~\ref{memone_group_best_response} which does not only have game theoretic
-novelty, but also the mathematical novelty of solving quadratic ratio optimisation
-problems where the quadratics are non concave.
+Theorem~\ref{theorem_one} has to a further theoretical result of identifying
```
I don't know what you are trying to say here.

Nikoleta-v3 replied:
There is one further theoretical result that can be obtained from Theorem~\ref{theorem_one}.

drvinceknight commented on main.tex:
```diff
@@ -631,34 +635,33 @@ \section{Discussion}
+In the interactions we have considered here the players do not make mistakes; their
+actions were executed with perfect accuracy. Mistakes, however, are relevant in
+the reasearch of repeated games~\cite{Boyd1989, Imhof2007, Nowak1993, Wu1995}. In
+future work we would consider interactions with ``noise''. Noise can be easily
```
```suggestion
future work we would consider interactions with ``noise''. Noise can be
```

drvinceknight committed:
Make minor changes

Take a look at these @nikoleta-v3 as I specifically remove one addition you made - we can chat about that during our meeting (the other changes I'm just fixing the language).

drvinceknight committed:
Update figure with latest data.

drvinceknight committed:
Add some summary statistics.

drvinceknight commented:
@Nikoleta-v3 there is now only a single TODO left in the paper (to archive the dataset).

drvinceknight commented on main.tex:
```diff
@@ -631,34 +634,33 @@ \section{Discussion}
-problems where the quadratics are non concave.
+Theorem~\ref{theorem_one} has to a further theoretical result of identifying
+environments for which cooperation cannot occur (Details are in the
+Appendix\ref{appendix:stability_of_defection}). Moreover,
```
```suggestion
There is one further theoretical result that can be obtained from Theorem~\ref{theorem_one}, 
which allows the identification of 
environments for which cooperation cannot occur (Details are in the
Appendix\ref{appendix:stability_of_defection}). Moreover,
```

Nikoleta-v3 committed:
Update main.tex

Co-authored-by: Vince Knight <vincent.knight@gmail.com>

drvinceknight commented on main.tex:
```diff
@@ -415,9 +421,9 @@ \subsection{Limitations of extortionate behaviour} %TODO check that this is a go
 response is not uniformly a ZDs. A positive measure of skewness and kurtosis,
 and a mean of 0.34 indicate a heavy tail to the right. Therefore, in several
-cases the strategy is not trying to extort its opponents.
+cases the strategy is not trying to extort its opponents. 
```
```suggestion
cases the strategy is not trying to extort its opponents. This highlights the importance of adaptability since the best response strategy against an opponent is rarely (if ever) a unique ZDs.
```

Nikoleta-v3 committed:
Update main.tex

Co-authored-by: Vince Knight <vincent.knight@gmail.com>

drvinceknight committed:
Update data and fix minor typos.

Nikoleta-v3 committed:
Fix citations (#79)

* Add data citation

* include citations to data to the main.tex

drvinceknight commented:
Merging as CI on #63 will cover anything.
